tests/scripts/t3000/run_t3000_unit_tests.sh

```diff
@@ -34,7 +34,6 @@ run_t3000_ttnn_tests() {
   echo "LOG_METAL: Running run_t3000_ttnn_tests"
   WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml ./build/test/ttnn/test_multi_device
   WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml ./build/test/ttnn/unit_tests_ttnn
-  pytest -n auto tests/ttnn/unit_tests/test_multi_device_trace.py ; fail+=$?
```

Comment:
Is this suppose to be removed?

Reply:
Yes, I'm doubling the number of tests instantiated by calling:
`WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto tests/ttnn/unit_tests/test_multi_device_trace.py ; fail+=$?`, so it made sense to remove some tests to keep CI runtime reasonable. The tests being removed are slightly redundant at this point.